api/awsauth/README.md

```diff
@@ -13,46 +13,28 @@ npm install -g serverless
 
 ## Deploy
 
-Currently we have a `dev` and `prod` stage.
+Currently we have a `dev` and `prod` stage deployed via Github Actions.
```

Comment:
"stage" ?  Is that a common term?  Maybe "environment"?

Reply:
ah stage was the term used in the serverless framework - can change this to environment

Reply:
Feel free to leave it if it's their terminology!